.travis.yml

```diff
@@ -29,8 +29,10 @@ node_js:
 
 ## Use this to prepare your build for testing
 ## e.g. copy database configurations, environment variables, etc.
-# before_script:
-#
+## http://stackoverflow.com/questions/12336566/travis-ci-with-jasmine-node
+before_script:
```

Comment:
Is this a travis issue?
Sounds like adding something to
after_script:
 or
after_success:

Is it just
npm start
 or
npm start <server-thingy> ?

On Fri, Jan 9, 2015 at 9:44 PM, Andrew McPherson <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/viditor/cumulonimbus/pull/8#discussion-diff-22755555>:
>
> > @@ -29,8 +29,10 @@ node_js:
> >
> >  ## Use this to prepare your build for testing
> >  ## e.g. copy database configurations, environment variables, etc.
> > -# before_script:
> > -#
> > +## http://stackoverflow.com/questions/12336566/travis-ci-with-jasmine-node
> > +before_script:
>
> Can we move this to the install section instead? Since we might put the npm
> start here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/viditor/cumulonimbus/pull/8/files#r22755555>.
>



-- 

John Angele’s Contact Info:
email: bgm946@mocs.utc.edu , jtangele@gmail.com ,
john-angele@mocs.utc.edu Facebook:
https://www.facebook.com/profile.php?id=100001292847932  Google+:
https://plus.google.com/114837189675232463068/posts  Google Docs:
https://docs.google.com/document/d/1uzkQ_ZI_s7u-hGt75vXZ5Bx46GM-hvxxK5PSwFc9ZZU/edit

Phone: (423) 280-0315 ; 4232800315@vtext.com (text cell from email)